src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -473,7 +478,7 @@ def test_resource_policy(self, resource_group):
 
         # create a policy assignment using a built in policy definition name
         self.kwargs['pan2'] = self.create_random_name('azurecli-test-policy-assignment2', 40)
-        self.kwargs['bip'] = self.cmd('policy definition list --query "[?policyType==\'BuiltIn\']|[0]"').get_output_in_json()['name']
+        self.kwargs['bip'] = '06a78e20-9358-41c9-923c-fb736d382a4d'
```

Comment:
what is this?

Reply:
The logic of querying the first BuiltIn policy was brittle and resulted in a BadRequest error. So I hard coded it to a built-in that would succeed.